modules/gridBidAdapter.js

```diff
@@ -362,6 +362,10 @@ function _addBidResponse(serverBid, bidRequest, bidResponses) {
         dealId: serverBid.dealid
       };
 
+      if (serverBid.ext && serverBid.ext.bidder && serverBid.ext.bidder.grid && serverBid.ext.bidder.grid.demandSource) {
```

Comment:
please copy demandSource to bid.meta.networkName . see https://github.com/prebid/prebid.github.io/pull/1911/files

Reply:
Please also duplicate this field server side; so that it will flow through regardless of if you are client or server post #6923

Reply:
#7184 should take care of client side